content/_meta/info/alert.md

```diff
@@ -1,9 +1,11 @@
 ---
-live: false
+live: true
 info_slug: alert
 title: alert
-aria: 'Black Lives Matter'
+aria: 'Subscribe to OddNews'
 ---
 
-**OddBird stands with the Black community in
-[demanding change](/community/blm/).**
+**Level up your CSS for free!**
+Grids, mixins,
+anchor positioning, and more.
+[Subscribe »](/oddnews/)
```

Comment:
Generally looks good, just some notes on wording

- I like the title/subtitle of the page itself
- Feels odd to say 'subscribe' then link to a page that says 'subscribe below'

An attempt

```suggestion
[Level up your CSS](/oddnews/)
with free cutting-edge demos in your inbox!
```